app/src/main/java/com/example/cooktaildb/ui/home_fragment/HomeFragment.kt

```diff
@@ -118,12 +120,18 @@ class HomeFragment : BaseFragment<FragmentHomeBinding>(FragmentHomeBinding::infl
         }
     }
 
-    private fun getDrinkBySelectedTab(strCategory: String?) {
-        strCategory?.let { homePresenter?.getDrinkBySelectedCategory(it) }
-    }
-
     companion object {
         @JvmStatic
         fun newInstance() = HomeFragment()
     }
+
+    override fun onItemClick(idDrink: String) {
+        context?.let {
+            DetailDrinkActivity.getIntent(it).also { intent ->
+                val bundle = Bundle()
+                bundle.putString(Constant.BUNDLE_ID_DRINK, idDrink)
```

Comment:
```suggestion
                val bundle = bundleOf(Constant.BUNDLE_ID_DRINK to idDrink)
```
Có thể tham khảo cách tạo này nhìn cho nó xịn xò :D